fleet-charts/gemma-server/values.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "properties": {
+    "hf_api_token": {
+      "description": "Hugging Face token",
+      "type": "string"
+    }
+  },
+  "required": [
+    "hf_api_token"
```

Comment:
how must this be set?

Reply:
This should be set as a Helm value.  For example is calling from TF:

```
resource "helm_release" "" {
  name  = 
  chart = 

  set_sensitive {
    name  = "hf_api_token"
    value = local.hf_api_token
  }
}
```

I haven't tested it yet, but when using in  an ApplicationSet, my understanding is we would set the value using the list generator as the first step: https://argo-cd.readthedocs.io/en/stable/user-guide/application-set/